Tests/TestAppUITests/TestAppUITests.swift

```diff
@@ -23,12 +28,47 @@ class TestAppUITests: XCTestCase {
             XCTAssertEqual(error as? ScreenObject.WaitForScreenError, .timedOut)
         }
     }
+
+    func testScreenInitThrowsWhenScreenWithMultipleElementsIsNotLoaded() throws {
+        do {
+            _ = try MissingMultipleElementsScreen(app: app)
+            XCTFail("Expected `ScreenObject` `init` to throw, but it didn't")
+        } catch {
+            XCTAssertEqual(error as? ScreenObject.WaitForScreenError, .timedOut)
+        }
+    }
+
+    func testInitThrowsWhenGivenEmptyGettersArray() throws {
+        do {
+            _ = try ScreenObject(expectedElementGetters: [], app: app)
+            XCTFail("Expected `ScreenObject` `init` to throw, but it didn't")
+        } catch {
+            XCTAssertEqual(error as? ScreenObject.InitError, .emptyExpectedElementGettersArray)
+        }
+    }
 }
 
-final class HelloWorldScreen: ScreenObject {
+class HelloWorldScreen: ScreenObject {
 
-    init(app: XCUIApplication = XCUIApplication()) throws {
-        try super.init(expectedElementGetter: { $0.staticTexts["Hello, world!"] })
+    init(app: XCUIApplication) throws {
+        try super.init(expectedElementGetter: { $0.staticTexts["Hello, world!"] }, app: app)
```

Comment:
This has nothing to do with this PR, it's just a cleanup that I though made sense to include while I was at it.

I prefer to inject the `XCUIApplication` instance rather than creating a new one, even though the result is the same: they both point to the test app used for these test.